libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -1857,6 +1858,215 @@ def produce_docs():
         f.write(doc_str)
 
 
+def get_std_dialects(data):
+    """Impementation for feature_test_macros.get_std_dialects()."""
+    dialects = set()
+    for feature in data:
+        keys = feature["values"].keys()
+        assert len(keys) > 0, "'values' is empty"
+        dialects |= keys
+
+    return sorted(dialects)
```

Comment:
```suggestion
    return sorted(list(dialects))
```

Because the attribute is going to return a `List[str]`.

Reply:
From what I understand this happens implicitly, that's why I left it out. I added it.